src/helpers/functions.test.ts

```diff
@@ -0,0 +1,64 @@
+import { type EventPayload } from "../types";
+import { parseFnData } from "./functions";
+import { InngestAPI } from "../api/api";
```

Comment:
In test files, these imports need to be prefixed with `@local`. This is a cheeky trick used so we can test the built types against multiple consuming versions of TypeScript.

If you don't do this, type tests will silently always use the library's version of TS (currently 5), meaning we're not actually testing for 4.7+ support.

This is a major footgun that needs a custom ESLint rule adding to `rules/`.

```suggestion
import { InngestAPI } from "@local/api/api";
import { parseFnData } from "@local/helpers/functions";
import { type EventPayload } from "@local/types";
```